CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contributing to Consul-ECS
+
+Thanks for your interest in Consul on ECS. We value feedback and contributions from the community whether it is a bug report, enhancements, new features or documentation.
+
+Please read through this document before submitting any issues or pull requests to ensure we have all the necessary 
+information to effectively respond to your bug report or contribution.
+
+
+## Reporting Issues, Bugs or Feature Requests
+
+We welcome users to report issues or suggest features.
+
+If you're suggesting a new feature (i.e., functionality that doesn't exist yet), please use our [issue template](https://github.com/hashicorp/consul-ecs/issues).  This will prompt you to answer a few questions that will help us figure out what you're looking for.  The template will also tag incoming issues with "Enhancement".  This gives us a way to filter the community-opened issues quickly so we can review as a team.
```

Comment:
Do we have any templates?

Reply:
Need to create a new template. Will create something on the lines of this https://github.com/hashicorp/consul-terraform-sync/issues/new?assignees=&labels=&template=feature_request.md